### PR TITLE
Fix injection on both OS

### PR DESCRIPTION
**Pull request by brunobar79**
Fixes APP-####

## What changed (plus any additional context for devs)
- Android wasn't working at all
- iOS broke after some UI refactor

## Screen recordings / screenshots
iOS: http://recordit.co/tUDH0i6Xj9
Android: http://recordit.co/WE1aWM6RUU

### CODE REVIEW

File: src/components/DappBrowser/BrowserTab.tsx [hunk 1]
```diff
@@ -262,9 +263,14 @@ export const BrowserTab = React.memo(function BrowserTab({ tabIndex, injectedJS
   const [backgroundColor, setBackgroundColor] = useState<string>();
 
   const createMessengers = useCallback((origin: string, tabId: string) => {
-    if (!webViewRef.current) return;
+    console.log('creating messengers');
```
Comment (skylarbarrera) [inline src/components/DappBrowser/BrowserTab.tsx:266]: boooooo

**Review by skylarbarrera — APPROVED** [review]
other than logs lgtm

File: src/components/DappBrowser/BrowserTab.tsx (hunk 1 above)
Comment (brunobar79) [reply src/components/DappBrowser/BrowserTab.tsx]: be careful bro https://github.com/search?q=repo%3Arainbow-me%2Frainbow+echo+%27hi&type=code

Comment (skylarbarrera) [reply src/components/DappBrowser/BrowserTab.tsx]: lmao

**Review by benisgold — APPROVED** [review]

**Pull request merged**